admin/modules/tools/tasks.php

```diff
@@ -393,8 +393,8 @@ function check_time_values($value, $min, $max, $return_type)
 
 	$form_container = new FormContainer($lang->edit_task);
 	echo $form->generate_hidden_field("tid", $task['tid']);
-	$form_container->output_row($lang->title." <em>*</em>", "", $form->generate_text_box('title', $task_data['title'], array('id' => 'title')), 'title');
-	$form_container->output_row($lang->short_description." <em>*</em>", "", $form->generate_text_box('description', $task_data['description'], array('id' => 'description')), 'description');
+	$form_container->output_row($lang->title." <em>*</em>", "", $form->generate_text_box('title', $task_data['title'] ?? null, array('id' => 'title')), 'title');
+	$form_container->output_row($lang->short_description." <em>*</em>", "", $form->generate_text_box('description', $task_data['description'] ?? null, array('id' => 'description')), 'description');
```

Comment:
This operator was [added in PHP 7.0](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op), so we'd need to use something else (e.g. assign all expected `$task_data` values individually using `$mybb->get_input()` instead of `$task_data = $mybb->input;`)

(one of the reasons why the lint test is failing)